packages/utils/src/lib/reports/scoring.ts

```diff
@@ -19,18 +19,18 @@ export function scoreReport(report: Report): ScoredReport {
   const allScoredAuditsAndGroups = new Map<string, AuditReport | ScoredGroup>();
 
   const scoredPlugins = report.plugins.map(plugin => {
-    const { slug, audits, groups } = plugin;
+    const { groups, ...pluginProperties } = plugin;
```

Comment:
"Props" is a common abbreviation for "properties" in code.

```suggestion
    const { groups, ...pluginProps } = plugin;
```